server/src/main/antlr4/org/eclipse/lsp/cobol/core/parser/CobolPreprocessor.g4

```diff
@@ -207,6 +209,18 @@ familyPhrase
    ;
 
 replaceable
+   : literal | cobolWord | charDataLine | functionCall
+   ;
+
+functionCall
```

Comment:
Why did you add it here?

Reply:
It's part of the syntax. We don't want to support this functionality but just the syntax (as discussed with Scott).
I am not sure it works. I will keep this conversation open, till I have a use-case for it or it works (syntactically)

![image](https://user-images.githubusercontent.com/69206564/110651005-559aae00-81bb-11eb-9381-8226d5099c57.png)

Reply:
covered with a use-case test.